bin/create-test-data.sh

```diff
@@ -10,7 +10,8 @@ echo -n "Determining provider bucket..."
 provider_bucket="$(
   echo 'var.buckets["provider"]["name"]' |
     terraspace console cumulus 2>/dev/null |
-    grep "${CUMULUS_PREFIX}"
+    grep "${CUMULUS_PREFIX}" |
+    sed -E 's/"(.+)"/\1/'
```

Comment:
```sh
$ make create-test-data

Determining provider bucket..."csda-cumulus-ali-provider-7894"

Syncing dummy files to provider bucket "csda-cumulus-ali-provider-7894"...fatal error: Parameter validation failed:
Invalid bucket name ""csda-cumulus-ali-provider-7894"": Bucket name must match the regex "^[a-zA-Z0-9.\-_]{1,255}$" or be an ARN matching the regex "^arn:(aws).*:(s3|s3-object-lambda):[a-z\-0-9]*:[0-9]{12}:accesspoint[/:][a-zA-Z0-9\-.]{1,63}$|^arn:(aws).*:s3-outposts:[a-z\-0-9]+:[0-9]{12}:outpost[/:][a-zA-Z0-9\-]{1,63}[/:]accesspoint[/:][a-zA-Z0-9\-]{1,63}$"
make: *** [create-test-data] Error 1
```

Reply:
Strange. I wonder if output changed with a recent version of terraform or terraspace, since this didn't have this problem before.

Reply:
That was my guess, but I couldn't (easily) find Terraspace change history to confirm. I actually aligned with existing handling [here](https://github.com/NASA-IMPACT/csdap-cumulus/blob/ea24aad28da55e5a347e776bde571a168d3061ec/bin/create-data-management-items.sh#L6-L10), so it might've changed around that time since the DevEx work surrounding the sandbox tickets looked pretty thoroughly tested 🤷

Reply:
Ah, yes, perhaps just something missed back then.